msteams-platform/toolkit/debug-your-Teams-app-test-tool.md

```diff
@@ -0,0 +1,478 @@
+    userPrincipleName: isaiah@example.com
+    aadObjectId: 00000000-0000-0000-0000-0000000000023
+    givenName: Isaiah
+    surname: Langer
+    email: isaiahl@example.com
+  - id: user-id-4
+    name: Patti Fernandez
+    userPrincipleName: pattif@example.com
+    aadObjectId: 00000000-0000-0000-0000-0000000000024
+    givenName: Patti
+    surname: Fernandez
+    email: pattif@example.com
+  - id: user-id-5
+    name: Lynne Robbins
+    userPrincipleName: lynner@example.com
+    aadObjectId: 00000000-0000-0000-0000-0000000000025
+    givenName: Lynne
+    surname: Robbins
+    email: lynner@example.com
+personalChat:
+  id: personal-chat-id
+groupChat:
+  id: group-chat-id
+  name: Group Chat
+team:
+  id: team-id
+  name: My Team
+  aadGroupId: 00000000-0000-0000-0000-000000000031
+  channels:
+    - id: channel-announcements-id
+      name: Announcements
+```
+
+</details>
+
+### Update the configuration file
+
+If your bot code uses Bot Framework APIs, you can modify the configuration file to alter the API response. Let us consider an Azure DevOps notification bot installed in a team that fetches inactive bugs from Azure DevOps. It identifies the owners of the inactive bug, fetches their email addresses, and sends notifications to their personal chats on a daily basis.
+
+To comprehensively test this bot in the Test Tool, it's crucial to update the configuration file to use the correct email addresses of the owners of the inactive bugs.
+
+1. Go to the `teamsapptesttool.yml` file in the project's root folder.
+
+1. Go to the `users` section and update the `name`, `userPrincipleName`, and `email` of the required user.
+
+    ```yaml
+    users:
+        - id: user-id-1
+          name: Megan Bowen
+          userPrincipleName: meganb@example.com
+          aadObjectId: 00000000-0000-0000-0000-0000000000021
+          givenName: Megan
+          surname: Bowen
+          email: some-real-user@real-domain.onmicrosoft.com
+    ```
+
+1. Save the file and select **F5** to debug in Test Tool.
+
+> [!NOTE]
+> When you edit the configuration file in Visual Studio Code, Intellisense will automatically update the property names and warn you if you enter invalid values.
+
+It's vital to recognize that updating the configuration file has three major impacts:
+
+* It influences the responses obtained from Bot Framework Connector APIs (example: `TeamsInfo.getPagedMembers()`).
+* It shapes the details in the activity payload (example: `activity.recipient`).
+* It impacts the user interface in the Test Tool (example: group chat names).
+
+## Limitations
+
+1. Bot features enabled through the Teams app manifest aren't available as The Test Tool doesn't process it.
+
+1. The Test Tool doesn't support all other types of bot application [Cards](../task-modules-and-cards/what-are-cards.md#cards).
+
+1. The Test Tool doesn't support the following Adaptive Card features:
+
+   * [People Picker](../task-modules-and-cards/cards/people-picker.md)
```

Comment:
People Picker is already supported, please remove it from the section.

Reply:
Done. Removed.